File: sample/C#/CSharp/Kabusapi_CancelOrder.cs
using System;
using System.Text;
using Newtonsoft.Json;
using System.Net.Http;

namespace CSharp_sample
{
    class Kabusapi_CancelOrder
    {
        static void Main(string[] args)
        {
            var obj = new
            {
                OrderId = "20200709A02N04712032",
            };
            var url = "http://localhost:18080/kabusapi/cancelorder";
            string token = GenerateToken.GetToken();
            try
            {
                var client = new HttpClient();
                var request = new HttpRequestMessage(HttpMethod.Put, url);
                request.Headers.Add("ContentType", "application/json");
                request.Headers.Add("X-API-KEY", token);
                request.Content = new StringContent(JsonConvert.SerializeObject(obj), Encoding.UTF8, "application/json");

                HttpResponseMessage response = client.SendAsync(request).Result;
                Console.WriteLine("{0} \n {1}", JsonConvert.DeserializeObject(response.Content.ReadAsStringAsync().Result), response.Headers);

            }
            catch (HttpRequestException e)
            {
                Console.WriteLine("{0} {1}", e, e.Message);
                Console.ReadKey();
            }
            catch (Exception ex)
            {
                Console.WriteLine("{0} {1}", ex, ex.Message);
                Console.ReadKey();
            }

            Console.ReadKey();
        }
    }
}

File: sample/Python/kabusapi_cancelorder.py
import urllib.request
import json
import pprint

obj = { 'OrderID': '20200709A02N04712032' }
json_data = json.dumps(obj).encode('utf8')

url = 'http://localhost:18080/kabusapi/cancelorder'
req = urllib.request.Request(url, json_data, method='PUT')
req.add_header('Content-Type', 'application/json')
req.add_header('X-API-KEY', 'ed94b0d34f9441c3931621e55230e402')

try:
    with urllib.request.urlopen(req) as res:
        print(res.status, res.reason)
        for header in res.getheaders():
            print(header)
        print()
        content = json.loads(res.read())
        pprint.pprint(content)
except urllib.error.HTTPError as e:
    print(e)
    content = json.loads(e.read())
    pprint.pprint(content)
except Exception as e:
    print(e)
